packages/docs-page/server/loaders/file-system.test.ts

```diff
@@ -0,0 +1,242 @@
+            "page": Array [
+              "extending-waypoint",
+              "creating-plugins",
+              "example-application",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "creating-plugins",
+              "testing",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "main-func",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "passing-values",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "authenticator",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "configurable",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "configurable-notify",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "builder",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "registry",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "platform",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "release-manager",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "destroy",
+            ],
+          },
+        },
+        Object {
+          "params": Object {
+            "page": Array [
+              "extending-waypoint",
+              "plugin-interfaces",
+              "default-parameters",
+            ],
+          },
+        },
+      ]
+    `)
```

Comment:
This feels like a big enough snapshot to maybe not be best suited for inline, but that's 100% a nit and doesn't make any difference if you feel differently